src/renderer/components/message/Message.tsx

```diff
@@ -542,6 +542,36 @@ export default function Message(props: {
         </div>
       </div>
     )
+  } else if (message.vcardContact) {
+    const { profileImage, color, displayName, addr } = message.vcardContact
+    const codepoint = displayName && displayName.codePointAt(0)
+    const initial = codepoint
+      ? String.fromCodePoint(codepoint).toUpperCase()
+      : '#'
+    content = (
+      <div className={styles.vcard}>
+        <div
+          className={classNames('avatar', styles.avatar)}
+          aria-label={displayName}
+        >
```

Comment:
I think it makes sense to use the same Avatar component as chatlist?

Reply:
Yes I agree, but then we have to change the line <img className='content' src={'file://' + avatarPath} /> there, since the profileImage is base64 encoded. Will do that later...